client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -1,329 +0,0 @@
-/**
- * External dependencies
- */
-import React, { PureComponent } from 'react';
-import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
-import classNames from 'classnames';
-import debugFactory from 'debug';
-import { first, indexOf, isEmpty, isEqual, last, map, merge } from 'lodash';
-import emailValidator from 'email-validator';
-
-/**
- * Internal dependencies
- */
-import QueryContactDetailsCache from 'calypso/components/data/query-contact-details-cache';
-import QueryTldValidationSchemas from 'calypso/components/data/query-tld-validation-schemas';
```

Comment:
It doesn't look like this is used anywhere else.

Reply:
I'm going to leave that for now because it seems like it could be important if we want to add more client-side validation in the future.